strongbox-web-core/src/main/java/org/carlspring/strongbox/rest/ConfigurationManagementRestlet.java

```diff
@@ -30,6 +31,7 @@
 @Component
 @Path("/configuration/strongbox")
 @Api(value = "/configuration/strongbox")
+@PreAuthorize("hasRole('ARTIFACTS_DELETE')")
```

Comment:
@AlexOreshkevich ,
I am not quite sure why you need the permissions to delete artifacts in this restlet. It's actually meant for handling the configuration of the server.